project/plugins.sbt

```diff
@@ -12,14 +12,13 @@ resolvers += Resolver.url("2m-sbt-plugin-releases", url("https://dl.bintray.com/
 addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.3.8")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.14")
 addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.2.8")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.2.9")
 addSbtPlugin("com.dwijnand" % "sbt-dynver" % "1.1.1")
-addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.9.1")
-addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.3.3")
+//addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.9.1")
```

Comment:
I would be happy to, but #574 seems to still be alive. My motivation behind commenting this out is that we don't pay the cost of loading that plugin, however still leave `git grep -i osgi` helpful in case someone wants to add OGSI support.